tests/linearizability/model.go

```diff
@@ -43,10 +43,12 @@ type EtcdResponse struct {
 }
 
 type EtcdState struct {
-	Key          string
-	Value        string
-	LastRevision int64
-	FailedWrite  *EtcdRequest
+	Key            string
+	PossibleValues []valueRevision
+}
+
+func (resp EtcdResponse) Equals(other EtcdResponse) bool {
+	return resp.GetData == other.GetData && resp.Revision == other.Revision && resp.Deleted == other.Deleted && resp.Err == other.Err
 }
```

Comment:
Do not need at all. You can just use `etcdResp1 == etcdResp2` directly. 
```suggestion
```

Reply:
Done, I was worried about cases where we add pointers, but let's not worry about this now.